src/components/ShoppingCart/dropDownButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { GrDown, GrUp } from "react-icons/gr";
+import { IconButton } from "@chakra-ui/react";
+import { Collapse } from "@chakra-ui/react";
+
+type quotationDetail = {
+  content: string;
+};
+const DropDownButton = ({ content }: quotationDetail) => {
+  const [show, setShow] = useState(false);
+
+  const handleToggle = () => setShow(!show);
+
+  const scroll = "scroll";
+  const hidden = "hidden";
+  const noWrap = "nowrap";
+  const normal = "normal";
+  const clip = "clip";
+  const ellipsis = "ellipsis";
+
+  return (
+    <div
+      style={{
+        width: "100%",
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "center",
+        alignContent: "center",
+        overflow: "auto",
+      }}
+    >
```

Comment:
From the initial shopping cart design, all text aline to the left, even if the content is short. e,g,
![CartQuotationQuestion](https://user-images.githubusercontent.com/87405682/192128925-726f257d-b110-45b6-8ee0-4bc0b80f1cc5.png)

But after the design of showing quotation detail, the content is centered. When the content is short the content will be shifted to the center of the container.

![Cartdesignproblem2](https://user-images.githubusercontent.com/87405682/192128952-3aacb352-5a63-4e91-a68a-ec4463b49c54.png)

It might be better to fix this problem.

Reply:
The text alignment was changed to justifyContent: "start". This branch was closed as it is renamed to cc-0008.

Reply:
> From the initial shopping cart design, all text aline to the left, even if the content is short. e,g, ![CartQuotationQuestion](https://user-images.githubusercontent.com/87405682/192128925-726f257d-b110-45b6-8ee0-4bc0b80f1cc5.png)
> 
> But after the design of showing quotation detail, the content is centered. When the content is short the content will be shifted to the center of the container.
> 
> ![Cartdesignproblem2](https://user-images.githubusercontent.com/87405682/192128952-3aacb352-5a63-4e91-a68a-ec4463b49c54.png)
> 
> It might be better to fix this problem.

The text alignment was changed to justifyContent: "start". This branch was closed as it is renamed to cc-0008.